core/src/main/java/io/th0rgal/oraxen/compatibilities/provided/worldedit/WorldEditHandlers.java

```diff
@@ -129,7 +129,7 @@ private FurnitureMechanic getFurnitureMechanic(@NotNull BaseEntity entity) {
                     bukkitValues = (Map<String, Tag>) tag.getValue().get("BukkitValues").getValue();
                 } catch (Exception ignored) {
                 }
-                if (bukkitValues == null) return null;
+                if (bukkitValues == null || bukkitValues.get("oraxen:furniture") == null) return null;
```

Comment:
We call bukkitValues.get("oraxen:furniture") twice in succession, creating a variable directly to avoid double recovery.